testinfra/test/test_modules.py

```diff
@@ -256,3 +256,11 @@ def test_ansible_module(TestinfraBackend, Ansible):
         assert variables["inventory_hostname"] == "debian_jessie"
         assert variables["ansible_user"] == "root"
         assert variables["group_names"] == ["ungrouped"]
+
+
+@all_images
+def test_mountpoint(MountPoint):
+    root_mount = MountPoint('/')
+    assert root_mount.exists
+    assert 'rw' in root_mount.options
+    assert root_mount.filesystem == "aufs"
```

Comment:
docker has other filesystem backend like overlay and zfs. Just checking the value is not empty should be enough.